test/plugin/scenarios/kafka-scenario/src/main/java/test/org/apache/skywalking/apm/testcase/kafka/controller/CaseController.java

```diff
@@ -200,5 +208,44 @@ public void run() {
             consumer.close();
         }
     }
+
+    public class ConsumerThread2 extends Thread {
```

Comment:
After you change this, there is an `expectedData.yaml` file, you should add the expected segment data in it.